developer-docs/bolt-server.md

```diff
@@ -250,9 +250,9 @@ Where `<TARGET>` is either:
 * A vmpooler VM. To use this, replace `<TARGET>` above with the hostname.
 * One of the containers brought up by the `docker-compose` in the `spec` directory. To use these, you'll want to:
 
-    * get the IP of the **bolt-server container** (not the target container):
+    * get the IP that the **bolt-server container** believes it is hosted on (for example the IP of the developer laptop hosting the bolt-server container):
```

Comment:
@lucywyman Does this make sense to you?

Reply:
I think so. If you can find a way to work in that it's the laptop IP on the bridge network between the host and container, I found that useful, but it also assume a lot of knowledge about container networking that I don't think we need to explain there.